cogs/musicevents.py

```diff
@@ -80,6 +82,18 @@ async def check_leave_voice(self, guild):
                 await player.stop()
                 await self.connect_to(guild.id, None)
 
+    async def leave_check(self):
+        if len(self.bot.lavalink.player_manager.players) != 0:
+            for player_id in self.bot.lavalink.player_manager.players:
+                await self.check_leave_voice(player_id)
```

Comment:
Either this function has to receive a guild object or the function has to be changed to take guild IDs instead of guild objects. Atm it raises an exception, ending the leave timer task.